addons/payment_payumoney/controllers/main.py

```diff
@@ -21,8 +21,15 @@ def payu_return(self, **post):
         a new session cookie. Therefore, the previous session and all related information will be lost, so it will lead
         to undesirable behaviors. This is the reason why `save_session=False` is needed.
         """
+        return werkzeug.utils.redirect('/payment/process')
```

Comment:
Why would you stop processing payment data received from the redirect request? We cannot do that in a stable version as it would break all existing PayUmoney integrations until they configure the webhook.

Reply:
As per my tests, the data was never getting posted at the time of redirect. So the following condition will be never true.
```
if post:
    request.env['payment.transaction'].sudo().form_feedback(post, 'payumoney')
```

Reply:
Could it be because you disabled the client-side response from PayU's backend while enabling the webhook? [As per the doc](https://developer.payumoney.com/redirect/):
> Making a payment using PayUmoney redirect checkout gives two types of payment response:
> - A client-side response on the Success or Failure URL (SURL/FURL)
> - A server to server callback on the Webhook, if set from the PayUmoney dashboard.

I such client-side response was never posted, no transaction made with this acquirer would ever be confirmed.